api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Comparator.java

```diff
@@ -30,10 +30,14 @@
 
   private ComparatorStepResult stepResult;
 
-  private final List<Operation> filters = new ArrayList<>();
+  private List<Operation> filters = new ArrayList<>();
 
   private String comment;
 
+  public void setFilters(List<Operation> filters) {
```

Comment:
Please keep the order of class members and put methods after fields declaration.